daal4py/sklearn/model_selection/_split.py

```diff
@@ -161,17 +161,31 @@ def _daal_train_test_split(*arrays, **options):
 
             arr_copy = d4p.get_data(arr)
             if not isinstance(arr_copy, list):
-                arr_copy = arr_copy.reshape((arr_copy.shape[0], n_cols), order='A')
+                arr_copy = arr_copy.reshape(
+                    (arr_copy.shape[0], n_cols), order='A')
             if isinstance(arr_copy, np.ndarray):
                 order = 'C' if arr_copy.flags['C_CONTIGUOUS'] else 'F'
                 train_arr = \
-                    np.empty(shape=(n_train, n_cols), dtype=arr_copy.dtype, order=order)
+                    np.empty(
+                        shape=(
+                            n_train,
+                            n_cols),
+                        dtype=arr_copy.dtype,
+                        order=order)

```

Comment:
This line breaks is make automatically with autopep8?

Reply:
Yes